mod/directory.php

```diff
@@ -111,9 +111,12 @@ function directory_content(App $a)
 			$photo = 'photo';
 		}
 
+		$entries = [];
+
 		while ($rr = DBA::fetch($r)) {
 			$itemurl= '';
 
+		while ($rr = DBA::fetch($r)) {
```

Comment:
This one confuses me. I don't think that you can easily double fetch the same resource.

Reply:
This looks like an artefact of cherry-picking/rebasing/merging across four different branches, thanks for the keen eye.